.github/workflows/katello_react_tests.yml

```diff
@@ -0,0 +1,41 @@
+name: Katello React Tests (non-blocking)
+
+on:
+  pull_request:
+    branches: [develop]
+
+defaults:
+  run:
+    working-directory: ./katello
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          path: ${{ github.workspace }}/katello
+      - name: Use Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: '12.x'
+      # We could update the postinstall action for foreman to look for an environment variable for plugin webpack dirs
+      # before kicking off the ruby script to find them, this would eliminate the ruby dep and running `npm i` in Katello.
+      - uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: 2.6
+      - name: Checkout Foreman repo
+        uses: actions/checkout@v2
+        with:
+          repository: theforeman/foreman
+          path: ${{ github.workspace }}/foreman # checkout Foreman and Katello as siblings, same as dev env
+          ref: develop
+      - name: Install Foreman dependencies
+        run: npm i
+        working-directory: ${{ github.workspace }}/foreman
+      - name: Install Katello dependencies
+        run: npm i
+      - name: Run Katello linting
+        run: npm run lint
```

Comment:
Do we actually need to lint katello for foreman changes?

Reply:
right, that doesn't seem necessary :thinking:

Reply:
right, since they are importing the rules from `foreman-js` we doesn't need to. :+1: